privacyidea/lib/policy.py

```diff
@@ -2083,6 +2102,18 @@ def get_static_policy_definitions(scope=None):
                 'type': 'bool',
                 'desc': _("If this is checked, the seed "
                           "will be displayed as text during enrollment.")
+            },
+            ACTION.CONFIRMACTION: {
+                'type': 'str',
+                'desc': _('When to ask the user to confirm their action with a popup dialog. If set to "difficult", '
+                          'only ask when the change would require some work to revert. Defaults to "severe".'),
```

Comment:
Before this PR, the only confirmation dialog is for removing a user. After this PR, the default level (if not set by a policy), is SEVERE (set in `postpolicy.py`). This will still ask before deleting a user, but also before changing settings for a token type, or a resolver,  as well as before deleting a policy, resolver, realm, CA connector, periodic task, server, gateway, or token.

Reply:
OK, I am happy with that.